srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Hooks/WooCommerce/FieldElements.php

```diff
@@ -61,11 +63,11 @@ public static function charList($username, $deleted = false): void {
         <br>
         <script>
             function setIcon() {
-                var charicon = document.getElementById("char-icon");
-                var classicon = document.getElementById("class-icon");
-                var selected = jQuery(this).find("option:selected");
+                const charicon = document.getElementById("char-icon");
+                const classicon = document.getElementById("class-icon");
+                let selected = jQuery(this).find("option:selected");
```

Comment:
I don't see that selected is used somewhere else so, you could even use `const` I think